frontend/src/styles/colors.ts

```diff
@@ -2,8 +2,12 @@
 
 const cornflowerBlue = '#7467EF';
 const white = '#FFFFFF';
+const lightGray = '#d9d9d9';
+const gray = '#8a8a8a';
 
 export const Colors = {
   primary: cornflowerBlue,
   onPrimary: white,
+  divider: lightGray,
+  gray,
```

Comment:
Kolor gray nie jest generyczny, nazwa koloru powinna odzwierciedlać to co dany kolor powinien robić (np. loading spinner, border, header, itp). Gray to jest konkretna wartość